docs_source_files/content/webdriver/http_proxies.de.md

```diff
@@ -57,7 +57,22 @@ with webdriver.Firefox() as driver:
 
   {{< / code-panel >}}
   {{< code-panel language="csharp" >}}
-// Need pr
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+public class ProxyTest{
+	public static void Main() {
+		ChromeOptions options = new ChromeOptions();
+		Proxy proxy = new Proxy();
```

Comment:
Hi @nainappa ,

Code looks good to me. But can you please `beautify/indent` code with 2 spaces just to match up with other code samples and utilise space.

Regards,
Harsha

Reply:
@Harsha509 Changed the indentation to 2 spaces. Please take a look when you have time.